src/System.Data.SqlClient/src/project.json

```diff
@@ -13,7 +13,7 @@
     "System.IO.FileSystem": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10",
+    "System.Net.Sockets": "4.0.10-beta-*",
```

Comment:
This change isn't showing up in my local version, and there are no other changes mentioned in its history.